ATP.DataAccessLayer.Test/RepositoryTest/BookingRepositoryTest.cs

```diff
@@ -8,24 +8,24 @@ namespace ATP.DataAccessLayerTest.RepositoryTest
 {
     public class BookingRepositoryTest
     {
+        private static string _csvFilePath = "test.csv";
         private static Mock<StreamReader> _streamReaderMock;
         private static Mock<CsvReader> _csvReaderMock;
 
         [Fact]
         public void ReadBookings_FromCsv_Success()
         {
             // Arrange
-            var csvFilePath = "test.csv";
             var csvData = "Id,FlightId,FlightClass,BookingDate,DepartureCountry,DestinationCountry\n1,1,Economy,2024-04-25T12:00:00Z,USA,UK\n";
-            File.WriteAllText(csvFilePath, csvData);
+            File.WriteAllText(_csvFilePath, csvData);
 
-            _streamReaderMock = new Mock<StreamReader>(csvFilePath);
+            _streamReaderMock = new Mock<StreamReader>(_csvFilePath);
             _csvReaderMock = new Mock<CsvReader>(_streamReaderMock.Object, CultureInfo.InvariantCulture);
```

Comment:
this is nearly the fourth time I say, move it to a constructor and use Fixture Freeze